### ISSUE TITLE
Neighbourhood admins can add news associated with their neighbourhoods

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#815]

**Comment by katjam** [comment]
@kimadactyl I moved this back to v1.0 since you commented yesterday that we might not need it in the way it is specified. We might even want to move it to the dream yard or close it as a won't fix? Up to you.

**Comment by kimadactyl** [comment]
yeah happy to leave this for now until we get a stronger use case for it but i think we will want it one day when the site has grown a bit more

**Comment by kimadactyl** [comment]
Superseded by the News v2 PRD in #3308 — article visibility now follows the partner (same edges as events), so neighbourhood admins reach news through their partners and we're not building an ArticleNeighbourhood relation.
